### PR TITLE
#788 Get tasks endpoint for projects

### LINKED ISSUE #788: Get Tasks Endpoint
as of now we don't need a new endpoint for this, we can just have the get project endpoints return the tasks

so that will just take updating the query args, updating the transformer, and updating the shared type

(you'll probably have to modify some test data too)

### PR DESCRIPTION
**Pull request by MWalve**
## Changes

Modified the shared types, the transformer and the query args to accommodate for tasks.

##Notes
MAKE SURE TO MERGE AFTER #789 is merged

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #788

### CODE REVIEW

**Review by Peyton-McKee — COMMENTED** [review]
Will approve once create task endpoint is merged, so that the only files changed are the ones you made changes on.

**Review by anthonybernardi — APPROVED** [review]
awesome work!

**Pull request merged**
